### PR TITLE
StackedBarChart and Stacked100BarChart Samples

### PR DESCRIPTION
**Pull request by igDivya**
StackedBarChart and Stacked100BarChart Samples

### CODE REVIEW

File: samples/charts/data-chart/stacked-100-bar-chart/App.razor [hunk 1]
```diff
@@ -0,0 +1 @@
+<Infragistics.Samples.Pages.Stacked100BarChart />
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/stacked-100-bar-chart/App.razor:1]: replace this line with content of Stacked100BarChart.razor file

File: samples/charts/data-chart/stacked-100-bar-chart/Pages/Stacked100BarChart.razor [hunk 2]
```diff
@@ -0,0 +1,62 @@
+﻿@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor;
+@inject IJSRuntime JSRuntime;
+
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/stacked-100-bar-chart/Pages/Stacked100BarChart.razor:4]: remove this whole file

File: samples/charts/data-chart/stacked-bar-chart/Pages/StackedBarChart.razor [hunk 3]
```diff
@@ -0,0 +1,77 @@
+﻿@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor;
+@inject IJSRuntime JSRuntime;
+
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/stacked-bar-chart/Pages/StackedBarChart.razor:4]: remove this whole file

File: samples/charts/data-chart/stacked-bar-chart/App.razor [hunk 4]
```diff
@@ -0,0 +1 @@
+<Infragistics.Samples.Pages.StackedBarChart/>
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/stacked-bar-chart/App.razor:1]: replace this line with content of StackedBarChart.razor file

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/data-chart/stacked-100-bar-chart/App.razor (hunk 1 above)
Comment (igDivya) [reply samples/charts/data-chart/stacked-100-bar-chart/App.razor]: done

File: samples/charts/data-chart/stacked-100-bar-chart/Pages/Stacked100BarChart.razor (hunk 2 above)
Comment (igDivya) [reply samples/charts/data-chart/stacked-100-bar-chart/Pages/Stacked100BarChart.razor]: done

File: samples/charts/data-chart/stacked-bar-chart/Pages/StackedBarChart.razor (hunk 3 above)
Comment (igDivya) [reply samples/charts/data-chart/stacked-bar-chart/Pages/StackedBarChart.razor]: done

File: samples/charts/data-chart/stacked-bar-chart/App.razor (hunk 4 above)
Comment (igDivya) [reply samples/charts/data-chart/stacked-bar-chart/App.razor]: done

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**